### ISSUE TITLE
Docs feedback: /alerting/unified-alerting/high-availability.md

### ISSUE THREAD

**Issue by j9t** [grafana/grafana#46438]
> https://grafana.com/docs/grafana/latest/alerting/unified-alerting/high-availability/

The ASCII art is hard to parse and understand, and probably close to inaccessible for anyone not able to view it.

Please consider a properly drawn graphic that has suitable alternative content (alternative text, or a description).

**Comment by achatterjee-grafana** [comment]
Thanks for the feedback. Working on it.
